gempy/core/data.py

```diff
@@ -1975,6 +1978,113 @@ def create_orientation_from_surface_points(self, surface_points: SurfacePoints,
 
         return np.array([*center, *orientation, *normal])
 
+    def create_orientation_from_NN(self, surface_points,  searchcrit):
```

Comment:
make NN in lower case, we are trying to be more strict with PEP

Reply:
Done!